edx_proctoring/api.py

```diff
@@ -899,10 +899,17 @@ def update_attempt_status(exam_id, user_id, to_status, raise_if_not_found=True,
             return_course_info=True
         )
 
-        send_proctoring_attempt_status_email(
-            exam_attempt_obj,
-            credit_state.get('course_name', _('your course'))
-        )
+        if credit_state:
+            send_proctoring_attempt_status_email(
+                exam_attempt_obj,
+                credit_state.get('course_name', _('your course'))
+            )
+        else:
+            log.error(
+                "Could not retrieve the credit state for the user with the user id %r in the course %r.",
```

Comment:
Discussed in person.  I thought this was the `credit_service` and not `credit_state`, so part of my comment doesn't make sense.  I would switch this to info log as discussed.